extern/storage-sealing/commit_batch.go

```diff
@@ -452,24 +456,43 @@ func (b *CommitBatcher) Stop(ctx context.Context) error {
 	}
 }
 
-func getSectorDeadline(curEpoch abi.ChainEpoch, si SectorInfo) time.Time {
-	deadlineEpoch := si.TicketEpoch + policy.MaxPreCommitRandomnessLookback
+func (b *CommitBatcher) getCommitCutoff(si SectorInfo) (time.Time, error) {
```

Comment:
We should consider changing this function to return epochs, which would make it much easier to do testing with (not in this PR tho)

Reply:
Yeah, agreed...adding a TODO in the code